dev/com.ibm.ws.cdi.1.2.weld/bnd.bnd

```diff
@@ -19,7 +19,9 @@ Bundle-Description:CDI 1.2 Weld Integration, version ${bVersion}
 -dsannotations: com.ibm.ws.cdi.impl.DefaultSecurityService
 
 Export-Package: com.ibm.ws.cdi.executor, \
-                com.ibm.ws.cdi.utils
+                com.ibm.ws.cdi.utils, \
+                com.ibm.ws.cdi.impl.weld, \
```

Comment:
i don't think you want to export this package from here. Put the proxy class in it's own new package and export that from here and the 2.0 version